business/iter8.go

```diff
@@ -0,0 +1,279 @@
+package business
+
+import (
+	"encoding/json"
+	"github.com/kiali/kiali/kubernetes"
+	"github.com/kiali/kiali/models"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	kube "k8s.io/client-go/kubernetes"
+	"github.com/kiali/kiali/config"
+	"k8s.io/client-go/rest"
+	"time"
+	"fmt"
+	"github.com/kiali/kiali/log"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	corev1 "k8s.io/api/core/v1"
+	iter8v1alpha1 "github.com/iter8-tools/iter8-controller/pkg/apis/iter8/v1alpha1"
+	kialiConfig "github.com/kiali/kiali/config"
+)
+
+const GroupName = "iter8.tools"
+const GroupVersion = "v1alpha1"
+
+type Iter8Service struct {
+	k8s kubernetes.IstioClientInterface
+	businessLayer *Layer
+}
+
+func (in *Iter8Service) GetIter8Info() (models.Iter8Info, error) {
```

Comment:
If I understand the intention of this method, it tries to respond if Iter8 is active or not isn't it ?
I think the current implementation creates only two constants (SchemeGroupVersion and iter8Kind) that don't perform any runtime check.
Perhaps this method could also check if Iter8 is enabled/disabled from config instead ?

Reply:
The intention is to check if the CRD exists in the cluster or not. Of course, this is not enough check to see if the Iter8 is enabled or not ( or running normal or not).  I will check more if all the components of iter8 are running.